buttercup.el

```diff
@@ -41,6 +41,56 @@
 
 (require 'cl-lib)
 (require 'buttercup-compat)
+(require 'format-spec)
+(require 'ert nil t)
```

Comment:
Does this really need ert loaded?

Reply:
This is mainly to get access to any ert explainers in order to give more informative messages for test failures. Note that it's a soft dependency: if ERT is unavailable for some reason, it will just fall back to the non-explainer-based failure message.